.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Thanks for your hard work, please ensure all items are complete before opening.
+
+- [ ] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://github.com/cloudant/nodejs-cloudant/blob/master/DCO1.1.txt)
+- [ ] You have added tests for any code changes
+- [ ] You have updated the [CHANGES.md](https://github.com/cloudant/nodejs-cloudant/blob/master/CHANGES.md)
+- [ ] You have completed the PR template below:
+
+## What
+
+What was changed, e.g. Added support for querying views.
```

Comment:
Might be worth making this example content a bit more relevant to the follow library, but it's not essential.